packages/dependency-royalty-management/contracts/MultiRoyaltyDistributor.sol

```diff
@@ -28,19 +28,18 @@ abstract contract MultiRoyaltyDistributor is IEIP2981, IMultiRoyaltyDistributor,
 
     /// @notice Query if a contract implements interface `id`.
     /// @param interfaceId the interface identifier, as specified in ERC-165.
-    /// @return `true` if the contract implements `id`.
+    /// @return isSupported `true` if the contract implements `id`.
     function supportsInterface(bytes4 interfaceId)
         public
         view
         virtual
         override(ERC165Upgradeable, IERC165)
-        returns (bool)
+        returns (bool isSupported)
     {
-        return
-            interfaceId == type(IEIP2981).interfaceId ||
+        isSupported = (interfaceId == type(IEIP2981).interfaceId ||
```

Comment:
There is no need to do this, you can still just return as before without assigning the value to the `isSupported` variable